### ISSUE TITLE
Unable to generate world

### ISSUE THREAD

**Issue by DBotThePony** [The-Fireplace/Fires-Random-Things#7]
```
[14:53:06] [Server thread/ERROR]: Encountered an unexpected exception
net.minecraft.util.ReportedException: Exception initializing level
	at net.minecraft.world.WorldServer.func_72963_a(WorldServer.java:887) ~[oo.class:?]
	at net.minecraft.server.integrated.IntegratedServer.func_71247_a(IntegratedServer.java:123) ~[chd.class:?]
	at net.minecraft.server.integrated.IntegratedServer.func_71197_b(IntegratedServer.java:156) ~[chd.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:484) [MinecraftServer.class:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_152]
Caused by: java.lang.NullPointerException
	at the_fireplace.frt.handlers.StructurePlacementManager.queueGeneration(StructurePlacementManager.java:95) ~[StructurePlacementManager.class:?]
	at the_fireplace.frt.worldgen.StructureGenerator.generate(StructureGenerator.java:50) ~[StructureGenerator.class:?]
	at the_fireplace.frt.worldgen.WorldGeneratorStructure.generate(WorldGeneratorStructure.java:47) ~[WorldGeneratorStructure.class:?]
	at net.minecraftforge.fml.common.registry.GameRegistry.generateWorld(GameRegistry.java:167) ~[GameRegistry.class:?]
	at net.minecraft.world.chunk.Chunk.func_186034_a(Chunk.java:1016) ~[axw.class:?]
	at net.minecraft.world.chunk.Chunk.func_186030_a(Chunk.java:977) ~[axw.class:?]
	at net.minecraft.world.gen.ChunkProviderServer.func_186025_d(ChunkProviderServer.java:157) ~[on.class:?]
	at net.minecraft.world.World.func_72964_e(World.java:309) ~[amu.class:?]
	at net.minecraft.world.World.func_175726_f(World.java:304) ~[amu.class:?]
	at net.minecraft.world.World.func_180495_p(World.java:910) ~[amu.class:?]
	at net.minecraft.world.World.func_175623_d(World.java:229) ~[amu.class:?]
	at net.minecraft.world.World.func_184141_c(World.java:209) ~[amu.class:?]
	at net.minecraft.world.WorldProvider.func_76566_a(WorldProvider.java:76) ~[aym.class:?]
	at net.minecraft.world.WorldServer.func_73052_b(WorldServer.java:943) ~[oo.class:?]
	at net.minecraft.world.WorldServer.func_72963_a(WorldServer.java:865) ~[oo.class:?]
	... 4 more
```

**Comment by The-Fireplace** [comment]
Can I see your full fml log from when that happened?

**Comment by DBotThePony** [comment]
[crash-2017-12-13_14.53.06-server.txt](https://github.com/The-Fireplace-Minecraft-Mods/Fires-Random-Things/files/1555992/crash-2017-12-13_14.53.06-server.txt)
?

`fml-log` didnt saved for that instance sorry

**Comment by The-Fireplace** [comment]
No, the file I'm looking for is in the logs folder, it will be called
either fml-client-latest.log or fml-server-latest.log, depending on if
you're in SinglePlayer or not.

**Comment by DBotThePony** [comment]
[fml-junk-earlystartup.log](https://github.com/The-Fireplace-Minecraft-Mods/Fires-Random-Things/files/1557598/fml-junk-earlystartup.log)
[fml-client-latest.log](https://github.com/The-Fireplace-Minecraft-Mods/Fires-Random-Things/files/1557599/fml-client-latest.log)
?

Earlystartup contains tracebacks

**Comment by The-Fireplace** [comment]
Oh, I meant the fml-client-latest right after the crash happens. This one doesn't appear to have any data about the crash, and the timestamp is different from the crash report. Could you cause the crash again, then send me the fml-client-latest.log right after?
Also, while your fml-junk-earlystartup did point out another bug to me, it won't be relevant to this issue.

**Comment by DBotThePony** [comment]
I launch a *singleplayer* game
Crash report related to fml logs at same timestamp

[crash-2017-12-14_08.03.52-server.txt](https://github.com/The-Fireplace-Minecraft-Mods/Fires-Random-Things/files/1557699/crash-2017-12-14_08.03.52-server.txt)
Integrated server log
[2017-12-14-1.log](https://github.com/The-Fireplace-Minecraft-Mods/Fires-Random-Things/files/1557707/2017-12-14-1.log)

its basically ALL info i have!
![infos](https://i.dbot.serealia.ca/2017/12/9567a5abd2_explorer_2017-12-14_09-00-36.png)

**Comment by The-Fireplace** [comment]
Ok, thank you. I'll try to fix it ASAP.

**Comment by DBotThePony** [comment]
It just crashes at point where it should say `Prepearing Spawn Area`, because world contains no chunks at all.
World type: Superflat with default settings

**Comment by The-Fireplace** [comment]
Fixed in 7.3.1
